Sofa/framework/Helper/src/sofa/helper/system/DynamicLibrary.cpp

```diff
@@ -126,7 +126,7 @@ void * DynamicLibrary::getSymbolAddress(Handle handle,
                     path = path.parent_path() / symlinkHandlePath;
                 }
             }
-            return path;
+            return std::filesystem::canonical(path);
```

Comment:
I was not aware of this "canonical" function when I did the getRealPath() 😅
IMO just use  `std::filesystem::canonical()` on the path directly (instead of the lambda)

And canonical() can throw so it would be wise to try/catch

Reply:
Thank you, done !